fastlane/Fastfile

```diff
@@ -163,8 +163,10 @@ lane :update_hybrid_common do |options|
   sh('yarn install --no-immutable')
   commit_current_changes(commit_message: 'Update yarn.lock')
 
+  push_to_git_remote(set_upstream: true)
+
```

Comment:
we're pushing before and after this line?

Reply:
Yes, the next function can fail if the pod is not distributed yet, so ideally we push before and after the pod is updated

Reply:
let's add a comment for it